viewing_party/party.py

```diff
@@ -1,23 +1,136 @@
 # ------------- WAVE 1 --------------------
 
+from ast import And
+from itertools import count
+from site import USER_BASE
+from typing import Counter
+import copy # I want to import copy module to use deepcopy
+'''I have chosen to use the deepcopy to ensure that I am not making
+changes to the original list pass through the functions.
+I am making changes to the new list for three(3) of the functions in this wave.
+However, as we know, if I only make a shallow copy, it will also change the 
+original list since the elements in both the original list and the new list
+are referencing the same ID number. By creating a deepcopy, I can ensure that as 
+the function iterates through the old list and changes to the new list,  the 
+changes will only affect the new list.'''
+
+
 def create_movie(title, genre, rating):
-    pass
+    new_movie = {}
+
+    new_movie["title"] = title
+    new_movie["genre"] = genre
+    new_movie["rating"] = rating
+
+    if new_movie["title"] == None or\
+        new_movie["genre"] == None or\
+            new_movie["rating"] == None:
+        return None     

```

Comment:
I'd suggest flipping the order of the conditions around to handle the special case (ie, checking if something is falsy) first, so that the code can quickly check and get out of there. This arrangement is referred to as a "guard clause" or a "guard condition." Notice that it lets us move the main focus of the function (creating a dictionary from the parameters) out of an indented block, letting it stand out more clearly.

Another thing is, if we flip the conditionals, the program doesn't waste memory space making a dictionary that won't be returned if the parameters turn out to be falsy

```suggestion
    if not title or genre or rating:
        return None
    new_movie = {}
    new_movie["title"] = title
    new_movie["genre"] = genre
    new_movie["rating"] = rating
```

We could also create the dictionary with all keys defined at once to dry up your code a little, but in this case it makes a miniscule difference